xSqlServerMaxDop: Measure-Resource value of Property is not valid

twerthi opened the pull request:
**Pull Request (PR) description**
- Changes to xSqlServerMaxDop
  - Fixed error where Measure-Object cmdlet would fail claiming it could not
    find the specified property ([issue #801](https://github.com/PowerShell/xSQLServer/issues/801))

**This Pull Request (PR) fixes the following issues:**
Fixes #801 
Fixes #805 

**Task list:**
<!--
To aid community reviewers in reviewing and merging your pull request (PR), please take
the time to run through the below checklist.
Change to [x] for each task in the task list that applies to your pull request (PR).
-->
- [x] Change details added to Unreleased section of CHANGELOG.md?
- [ ] Added/updated documentation, comment-based help and descriptions in .schema.mof files where appropriate?
- [ ] Examples appropriately updated?
- [ ] New/changed code adheres to [Style Guidelines](https://github.com/PowerShell/DscResources/blob/master/StyleGuidelines.md)?
- [ ] [Unit and (optional) Integration tests](https://github.com/PowerShell/DscResources/blob/master/TestsGuidelines.md) created/updated where possible?

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/powershell/xsqlserver/802)
<!-- Reviewable:end -->

msftclas commented:
@__twerthi__,
Thanks for your contribution.
To ensure that the project team has proper rights to use your work, please complete the Contribution License Agreement at https://cla.microsoft.com.

It will cover your contributions to all Microsoft-managed open source projects.
_Thanks,
Microsoft Pull Request Bot_

msftclas commented:
__@twerthi__, thanks for signing the contribution license agreement. We will now validate the agreement and then the pull request.
<br />_Thanks, Microsoft Pull Request Bot_

twerthi commented:
Build indicates that CHANGELOG.md failed, I'm not entirely sure what I did wrong.  Would someone be able to point out the issue and I'll fix it?

randomnote1 commented:
Review status: 0 of 2 files reviewed at latest revision, 1 unresolved discussion, some commit checks failed.

---

*[CHANGELOG.md, line 20 at r1](https://reviewable.io:443/reviews/powershell/xsqlserver/802#-KtnR3bQFNVY1UtU3jBR:-KtnR3bQFNVY1UtU3jBS:b-fxiti) ([raw file](https://github.com/powershell/xsqlserver/blob/e18a1f186012a97e0cce2b768dc6fae0104aac14/CHANGELOG.md#L20)):*
> ```Markdown
>     Members is not an array ([issue #790](https://github.com/PowerShell/xSQLServer/issues/790)).
> - Changes to xSqlServerMaxDop
>   - Fixed error where Measure-Object cmdlet would fail claiming it could not find the specified property ([issue #801](https://github.com/PowerShell/xSQLServer/issues/801))
> ```

This failed the tests because it is longer than 80 characters in length excluding the issue reference. If you put a new line before "find", it should pass cleanly.

```
  - Fixed error where Measure-Object cmdlet would fail claiming it could not
    find the specified property ([issue #801](https://github.com/PowerShell/xSQLServer/issues/801))
```

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/powershell/xsqlserver/802)*
<!-- Sent from Reviewable.io -->

twerthi commented:
Ah!!!  Thank you, I will fix immediately.

Shawn Sesna
DevOps Engineer
Department of Early Learning
P.O. Box 40970 | 505 Union Ave. S.E., Suite 300
Shawn.sesna@del.wa.gov<mailto:Shawn.sesna@del.wa.gov> | Cell: 360.628.7202
www.del.wa.gov<http://www.del.wa.gov/>
Kids' Potential, Our Purpose

From: Dan Reist [mailto:notifications@github.com]
Sent: Monday, September 11, 2017 5:18 PM
To: PowerShell/xSQLServer <xSQLServer@noreply.github.com>
Cc: Sesna, Shawn (DEL) <Shawn.Sesna@del.wa.gov>; Mention <mention@noreply.github.com>
Subject: Re: [PowerShell/xSQLServer] Fixed issue #801 (#802)


Review status: 0 of 2 files reviewed at latest revision, 1 unresolved discussion, some commit checks failed.

________________________________

CHANGELOG.md, line 20 at r1<https://reviewable.io:443/reviews/powershell/xsqlserver/802#-KtnR3bQFNVY1UtU3jBR:-KtnR3bQFNVY1UtU3jBS:b-fxiti> (raw file<https://github.com/powershell/xsqlserver/blob/e18a1f186012a97e0cce2b768dc6fae0104aac14/CHANGELOG.md#L20>):

    Members is not an array ([issue #790](https://github.com/PowerShell/xSQLServer/issues/790)).

- Changes to xSqlServerMaxDop

  - Fixed error where Measure-Object cmdlet would fail claiming it could not find the specified property ([issue #801](https://github.com/PowerShell/xSQLServer/issues/801))

This failed the tests because it is longer than 80 characters in length excluding the issue reference. If you put a new line before "find", it should pass cleanly.

  - Fixed error where Measure-Object cmdlet would fail claiming it could not

    find the specified property ([issue #801](https://github.com/PowerShell/xSQLServer/issues/801))

________________________________

Comments from Reviewable<https://reviewable.io:443/reviews/powershell/xsqlserver/802>

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/PowerShell/xSQLServer/pull/802#issuecomment-328694430>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AFS5RQMw8PcTifQ5jQPF5G65z_9sS5ghks5shc2pgaJpZM4PT4pu>.

johlju commented:
Reviewed 1 of 2 files at r1, 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 4 unresolved discussions.

---

*[DSCResources/MSFT_xSQLServerMaxDop/MSFT_xSQLServerMaxDop.psm1, line 268 at r2](https://reviewable.io:443/reviews/powershell/xsqlserver/802#-KtqubYB_v-O33ZnIlwe:-KtqubYB_v-O33ZnIlwf:b42aall) ([raw file](https://github.com/powershell/xsqlserver/blob/6d4ec3148b84c8433054513b0e80de9f16a29e59/DSCResources/MSFT_xSQLServerMaxDop/MSFT_xSQLServerMaxDop.psm1#L268)):*
> loop

Could you please change to 'Loop' (upper 'L')

---

*[DSCResources/MSFT_xSQLServerMaxDop/MSFT_xSQLServerMaxDop.psm1, line 269 at r2](https://reviewable.io:443/reviews/powershell/xsqlserver/802#-KtquOCC71ZLipiYqQni:-KtquOCC71ZLipiYqQnj:boypffr) ([raw file](https://github.com/powershell/xsqlserver/blob/6d4ec3148b84c8433054513b0e80de9f16a29e59/DSCResources/MSFT_xSQLServerMaxDop/MSFT_xSQLServerMaxDop.psm1#L269)):*
> $Processor

Please change to `$processor` (lower case 'p'). Throughout.

---

*[DSCResources/MSFT_xSQLServerMaxDop/MSFT_xSQLServerMaxDop.psm1, line 269 at r2](https://reviewable.io:443/reviews/powershell/xsqlserver/802#-KtquVw1dssKFHqAaHB4:-KtquVw1dssKFHqAaHB5:b-jcs206) ([raw file](https://github.com/powershell/xsqlserver/blob/6d4ec3148b84c8433054513b0e80de9f16a29e59/DSCResources/MSFT_xSQLServerMaxDop/MSFT_xSQLServerMaxDop.psm1#L269)):*
> foreach(

please add a space between the keyword and parenthesis 

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/powershell/xsqlserver/802)*
<!-- Sent from Reviewable.io -->

johlju commented:
@randomnote1 Could you please sign off on the review comment if you are satisfied (so it gets resolved).

randomnote1 commented:
@johlju Do I need to enter "lgtm"? Or just acknowledge?

johlju commented:
Hmm. You can just acknowledge if you just reviewed one file. If you reviewed the whole PR and you think it looks good to you, then type LGTM. 
Thanks for reviewing and helping!

johlju commented:
Even if here are comments from me or someone else, you can write LGTM. That means to me that you think the rest of the code that does not have review comments looks good :) It helps me. :)

randomnote1 commented:
Ok, I just looked at that one file to help through the error. I did not give the rest of the code a careful read-through.

twerthi commented:
All changes made :)

johlju commented:
@randomnote1 No worries! Thanks a lot for helping out! 😄

johlju commented:
@twerthi Could you please go into Reviewable, the big purple button in the PR description. In Reviewable please type 'Done' (or click on the Done-button) on each review comment. Do that on all review comments you have fixed, after that go to the top of the page and click on the big green Publish-button. That will take all your comments and publish them here in the PR.
When you write 'Done' on them, I can resolve them.
If you want to discuss a comment further, then just comment in Reviewable as usual.

twerthi commented:
Review status: 1 of 2 files reviewed at latest revision, 3 unresolved discussions.

---

*[DSCResources/MSFT_xSQLServerMaxDop/MSFT_xSQLServerMaxDop.psm1, line 268 at r2](https://reviewable.io:443/reviews/powershell/xsqlserver/802#-KtqubYB_v-O33ZnIlwe:-KtvqKK99VB93RFX7Cng:b-896fix) ([raw file](https://github.com/powershell/xsqlserver/blob/6d4ec3148b84c8433054513b0e80de9f16a29e59/DSCResources/MSFT_xSQLServerMaxDop/MSFT_xSQLServerMaxDop.psm1#L268)):*
<details><summary><i>Previously, johlju (Johan Ljunggren) wrote…</i></summary><blockquote>

> loop

Could you please change to 'Loop' (upper 'L')
</blockquote></details>

Done.

---

*[DSCResources/MSFT_xSQLServerMaxDop/MSFT_xSQLServerMaxDop.psm1, line 269 at r2](https://reviewable.io:443/reviews/powershell/xsqlserver/802#-KtquOCC71ZLipiYqQni:-KtvqKiAKmQrQXnjiyCD:b-896fix) ([raw file](https://github.com/powershell/xsqlserver/blob/6d4ec3148b84c8433054513b0e80de9f16a29e59/DSCResources/MSFT_xSQLServerMaxDop/MSFT_xSQLServerMaxDop.psm1#L269)):*
<details><summary><i>Previously, johlju (Johan Ljunggren) wrote…</i></summary><blockquote>

> $Processor

Please change to `$processor` (lower case 'p'). Throughout.
</blockquote></details>

Done.

---

*[DSCResources/MSFT_xSQLServerMaxDop/MSFT_xSQLServerMaxDop.psm1, line 269 at r2](https://reviewable.io:443/reviews/powershell/xsqlserver/802#-KtquVw1dssKFHqAaHB4:-KtvqLOOl3z6fVScqMuQ:b-896fix) ([raw file](https://github.com/powershell/xsqlserver/blob/6d4ec3148b84c8433054513b0e80de9f16a29e59/DSCResources/MSFT_xSQLServerMaxDop/MSFT_xSQLServerMaxDop.psm1#L269)):*
<details><summary><i>Previously, johlju (Johan Ljunggren) wrote…</i></summary><blockquote>

> foreach(

please add a space between the keyword and parenthesis 
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/powershell/xsqlserver/802)*
<!-- Sent from Reviewable.io -->

twerthi commented:
@johlju, thank you for the walk-through on that.

johlju commented:
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

No worries. Reviewable is not easy to use in the beginning :) I've been there. :smile:

---

Reviewed 1 of 1 files at r3.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/powershell/xsqlserver/802#-:-KtvsfxvXZ1xpHBUuuKm:b-njyjpp)*
<!-- Sent from Reviewable.io -->

johlju commented:
@twerthi Thanks a lot for this PR! Awesome work! 😄

twerthi commented:
My pleasure ☺

Shawn Sesna
DevOps Engineer
